ert_shared/main.py

```diff
@@ -465,6 +466,37 @@ def log_config(config_path: str, logger: logging.Logger) -> None:
         )
 
 
+@atexit.register
+def log_process_usage():
+    try:
+        import resource
+
+        usage = resource.getrusage(resource.RUSAGE_SELF)
+
+        if sys.platform == "darwin":
+            # macOS apparently outputs the maxrss value as bytes rather than
+            # kilobytes as on Linux.
+            #
+            # https://stackoverflow.com/questions/59913657/strange-values-of-get-rusage-maxrss-on-macos-and-linux
+            rss_scale = 1000
+        else:
+            rss_scale = 1
+
+        logging.info(f"User time: {usage.ru_utime} seconds")
```

Comment:
for Azure-logger, should we add this as one big dict instead and add to extras? It then ends up in properties of a log .,

Reply:
The azure logger uploader runs in the same process as ERT so I guess it can just retrieve that information itself?

Reply:
Not sure I understand @pinkwah . Who can return what itself?

Reply:
The problem with that is we need to to alot of parsing on the azure side of things, which is not super nice. It is better to have it all in one single statement (as extras), much easier to get the telemetry out that way.